.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,29 @@
-#### What is the purpose of this pull request?
-<!--- Describe your changes in detail. -->
-
 #### What problem is this solving?
+
 <!--- What is the motivation and context for this change? -->
 
 #### How should this be manually tested?
 
+[Workspace](url)
+
+#### Checklist/Reminders
+
+- [ ] Updated `README.md`.
+- [ ] If applicable, link this PR to a Clubhouse story.
+- [ ] Whenever possible, create tests for bug fixes.
```

Comment:
This item doesn't imply a "yes/no" answer because it has both a conditional and optional characteristic in it ("whenever posssible" and "if bug fix")